src/containers/shared/components/Transaction/EscrowCreate/parser.ts

```diff
@@ -0,0 +1,17 @@
+import { unix } from 'moment'
+import formatAmount from '../../../../../rippled/lib/txSummary/formatAmount'
+
+const EPOCH_OFFSET = 946684800
+const convertRippleDate = (date: any) =>
+  unix(date + EPOCH_OFFSET)
+    .utc()
+    .format()
```

Comment:
Probably worth pulling out into a helper method in a util file (Or I think xrpl.js has this built-in as well, but might not come out in exactly the desired format)